### DISCUSSION TITLE
The focused class not being removed when setting new item

### DISCUSSION THREAD

**Discussion by iamtylerd** [dowjones/react-dropdown-tree-select#623] [Q&A]
We have a picklist array with values used for populating the list.  We have the ability to add a new value a couple of parents up.  After adding a new value, that value is selected and the entire list is sent back to the dropdown.  The dropdown renders the new value and selects it (looking at the data it is the only one checked).  This is all correct, but the initial value I selected in the dropdown still has the focused class (not the checked class, expected behavior).  Any thoughts or suggestions on how to remove that class?  It seems to be only updating `onClick` not when new data is being passed down.

More info
We have a HOC that uses `useEffect` for the data prop changing and passing that data to the dropdown, I can see when adding a new value to data the lifecycle method is firing and data is different.

**Comment by mrchief** [comment]
I think it's a side effect of browser interaction with the checkboxes. Taking a blind guess, if you trick the checkbox to re-render as a new control, it should go away
